src/autora/doc/pipelines/data.py

```diff
@@ -0,0 +1,21 @@
+from typing import Iterable, List, Tuple
+
+
+def load_data(data_file: str) -> Tuple[List[str], List[str]]:
+    import jsonlines
```

Comment:
is this function only used once or multiple times? if the latter, probably better to move this to a top import

Reply:
ah, this is because jsonlines is an optional dependency used only for certain code paths so I only want to import it in the code paths where its actually called